disable-updates.php

```diff
@@ -237,6 +237,10 @@ public function block_request($pre, $args, $url) {
 		}
 
 		$url_data = parse_url( $url );
+		
+		if( !array_key_exists( 'path', $url_data ) ) {
+			return $pre;
+		}
```

Comment:
I also see this notice intermittently, here's a traceback https://sentry.io/share/issue/d4573581ed31497a8ea798335116d899/

In this case, `$url` value is pathless `https://api.wordpress.org`, which looks like a solid enough reason for this notice to occur. Your thoughts @oliverschloebe?

Reply:
Yeah true. Just returning `$pre` would just bypass all pathless requests to `https://api.wordpress.org` if the `path` key is missing. Let's see how to handle that. 🤔